app/lib/contentdm/importer.rb

```diff
@@ -22,8 +23,13 @@ def self.import
     def import
       @records.each do |record|
         begin
-          process_record(record)
+          work = process_record(record)
+          @log.info Rainbow("Adding #{work.id} to collection: #{collection_name}")
+          @collection.add_members(work.id)
```

Comment:
Can you add a spec that checks that the resulting works are members of the expected collection after the importer runs?  You could either check `work.in_collections` or `collection.members` in the spec.